### ISSUE TITLE
traceback2 does not conform to PEP 396 (missing __version__ string)

### ISSUE THREAD

**Issue by ddriddle** [testing-cabal/traceback2#2]
For our package management we use automated tooling to extract version numbers from python packages. I would like to request that you add the `__version__` attribute version string.

[PEP 396](https://www.python.org/dev/peps/pep-0396/) states:
- On a case-by-case basis, standard library modules which are also released in standalone form for other Python versions MAY include a module version number when included in the standard library, and SHOULD include a version number when packaged separately.
- When a module (or package) includes a version number, the version SHOULD be available in the `__version__` attribute.
- The `__version__` attribute's value SHOULD be a string.

**Comment by ddriddle** [comment]
Submitted pull request #3 to address this issue.

**Comment by berkerpeksag** [comment]
Thank you for the report and for the PR. Note that PEP 396 is not an accepted PEP and we recently got rid of several `__version__` attributes in both standard library and their backports (see email for example) Also, some of the recent standard library modules and their backports don't even have `__version__` attributes. See selectors34 for example.

And usually importing the module itself in `setup.py` is a bad practice.

Closing this issue as 'rejected'.
